client.lua

```diff
@@ -36,7 +36,7 @@ CreateThread(function()
         while ( not HasModelLoaded( GetHashKey(PedModel) ) ) do
             Wait(1)
         end
-        TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, true, true)
+        TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, false, false)
```

Comment:
`TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, false, false)`

I suggest that you change it to this

`TRClassicBlackMarketPed = CreatePed(1, PedHash, Coords, false, true)`

If you are able to test this change and let me know if it works then I will push the PR into the main resource 😄